source/game/field/ObjectDirector.cc

```diff
@@ -1,7 +1,10 @@
 #include "ObjectDirector.hh"
 
+#include "game/field/obj/ObjectDokan.hh"
+#include "game/field/obj/ObjectNoImpl.hh"
+#include "game/field/obj/ObjectOilSFC.hh"
```

Comment:
This is going to blow up in size as we add objects. Rather than have a million includes in the source file for an important class that we will likely have to debug in the future, could it instead make sense to have some sort of "helper header" in the `obj` folder which has the includes for all relevant objects? And then we just include that one header?

Reply:
Fixed with `ObjectRegistry.hh`.